draft-shafranovich-rfc4180-bis.md

```diff
@@ -141,7 +141,7 @@ However, some implementations MAY use other values.
 The ABNF grammar (as per {{!RFC5234}}) appears as follows:
 
 ~~~~~~~~~~
-file = [header [CR]LF] record *([CR]LF record) [ [CR]LF ]
+file = [header [CR]LF] 1*(record [CR]LF)
```

Comment:
Wondering about "1*". I think it would be possible to have a CSV file with no records, although that kind of defeats the grammar. Maybe:

`file = [header [CR]LF] *(record [CR]LF)`

Reply:
I had the same doubts but I didn't want to change the *existing* semantic.

I'm fine with `file = [header [CR]LF] *(record [CR]LF)` – it is more generous than RFC 4180.

Shall I update it accordingly?

Reply:
I think being a little lenient here would make sense, please update